### ISSUE TITLE
when will you release the code?

### ISSUE THREAD

**Issue by chenrxi** [FlorinShum/ULAST#1]
Thank you for work and when will you release the code?

**Comment by florinshen** [comment]
Thanks. I will release inference code and raw results recently, training code also will be available.

**Comment by chenrxi** [comment]
How do you generate the training data? In the paper, it said 'use unsupervised trained optical flow model [26] to generate the pseudo labels of the first frame'. It is a little bit confused. Can you give more details? Thank you. @FlorinShum

**Comment by florinshen** [comment]
Pseudo-label generation is the same as another work USOT, you can refer to this work for more details. The only difference is that our work samples one template and three search frames from a single video.

**Comment by chenrxi** [comment]
I see, thank you. I ignore the experimental detail part. What do you think why the model can improve robustness compared with USOT?

**Comment by chenrxi** [comment]
In Network architecture it says: ATSS [44] is used to assign anchor label with the top-15 candidates. There are K = 3125 candidate boxes in the prediction result of RPN. What does it mean.
